rules/endpoints.go

```diff
@@ -308,13 +308,15 @@ func (r *DefaultRuleRenderer) endpointIptablesChain(
 		},
 	})
 
-	if dropEncap {
+	if dropEncap && r.Config.DropVXLANPacketsFromWorkloads {
```

Comment:
Maybe the parameter should be consulted higher up the stack so that we only set dropEncap if we're rendering a workload egress chain and the flag is set?

Reply:
The reason why I did it this way was to add individual knobs to each type of encap. I could perhaps modify the type or add an additional parameter to `endpointIptablesChain` to distinguish IPIP versus VXLAN. WDYT?

Reply:
I've moved this up and tried to clean up a bit based on Casey's feedback. Let me know if this works.